### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by LBHBReynolds-Carr**

### CODE REVIEW

File: SocialCareCaseViewerApi/V1/UseCase/ProcessDataUseCase.cs [hunk 1]
```diff
@@ -17,11 +17,16 @@ public ProcessDataUseCase(IProcessDataGateway processDataGateway)
         }
         public CareCaseDataList Execute(ListCasesRequest request)
         {
+            request.Limit = request.Limit < 10 ? 10 : request.Limit;
+            request.Limit = request.Limit > 100 ? 100 : request.Limit;
```
Comment (LBHTKarki) [inline SocialCareCaseViewerApi/V1/UseCase/ProcessDataUseCase.cs:21]: Please add to the swagger doc that the max limit will be 100.

File: SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs [hunk 2]
```diff
@@ -79,7 +77,11 @@ public IEnumerable<CareCaseData> GetProcessData(ListCasesRequest request)
                 throw new DocumentNotFoundException("document not found");
             }
 
-            var response = ResponseFactory.ToResponse(result);
+            var response = ResponseFactory
```
Comment (LBHTKarki) [inline SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs:80]: For accurate limit and cursor values please set them after date filters have been applied

File: SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs [hunk 3]
```diff
@@ -62,15 +61,14 @@ public IEnumerable<CareCaseData> GetProcessData(ListCasesRequest request)
                     Builders<BsonDocument>.Filter.Regex("worker_email", BsonRegularExpression.Create(new Regex(request.WorkerEmail, RegexOptions.IgnoreCase))) : null;
                 var caseNoteTypeFilter = !string.IsNullOrWhiteSpace(request.CaseNoteType) ?
                     Builders<BsonDocument>.Filter.Regex("case_note_type", BsonRegularExpression.Create(new Regex(request.CaseNoteType, RegexOptions.IgnoreCase))) : null;
+                var sort = Builders<BsonDocument>.Sort.Ascending("first_name");
```
Comment (LBHTKarki) [inline SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs:64]: Please add the default sort column to swagger doc

**Review by LBHTKarki — CHANGES_REQUESTED** [review]
Please check that the cursor and limit are applied against correct result set

**Review by LBHBReynolds-Carr — COMMENTED** [review]
Updated with changes requested

**Review by LBHTKarki — APPROVED** [review]
Looks good

**Pull request merged**